pkg/controllers/cluster/resource/resource.go

```diff
@@ -350,6 +350,37 @@ func StatefulSetForRack(r scyllav1alpha1.RackSpec, c *scyllav1alpha1.ScyllaClust
 	if sysctlContainer != nil {
 		sts.Spec.Template.Spec.InitContainers = append(sts.Spec.Template.Spec.InitContainers, *sysctlContainer)
 	}
+
+	if c.Spec.Timezone != "" {
+		var timezone string
+		hostPath := corev1.HostPathFile
+		if c.Spec.Timezone[0] == '/' {
+			timezone = c.Spec.Timezone
+		} else {
+			timezone = "/usr/share/zoneinfo/" + c.Spec.Timezone
+		}
+
+		sts.Spec.Template.Spec.Containers[0].VolumeMounts = append(
+			sts.Spec.Template.Spec.Containers[0].VolumeMounts,
+			corev1.VolumeMount{Name: "host-timezone", MountPath: "/etc/localtime", ReadOnly: true})
+
+		// /etc/localtime is getting overridden by tmpfiles whenever scylla container is started.
```

Comment:
So if above is overwriten why do we mount it in first place?

Reply:
I think that rewriting of the timezone is result of a leftovers that were not removed from scylla docker image by mistake.
But despite that there are two reasons to do that:
1. It does not break anything in current docker image
2. It will make whole thing work when/if scylla image get rid of tmpfiles